Integrations/integration-GoogleResourceManager.yml

```diff
@@ -0,0 +1,1020 @@
+category: IT Services
+commonfields: {id: Google Resource Manager, version: -1}
+configuration:
+- defaultvalue: ''
+  display: The Project ID of the project resource in which you created the Service
```

Comment:
Change all parameters display names to be like `Project ID` - no need to put all this description here